.cirrus.yml

```diff
@@ -105,6 +105,9 @@ task:
     - name: Linux i686 musl
       env:
         TARGET: i686-unknown-linux-musl
+    - name: Linux LoongArch64
```

Comment:
Please add this here, Nix has switched to GitHub action for most CI checks:

 https://github.com/nix-rust/nix/blob/50e4283b35f3f34e138d138fd889f7e3c424a5c2/.github/workflows/ci.yml#L94